test/util.cr

```diff
@@ -55,6 +55,7 @@ STRIPPED_NODES = [
 # These nodes will be renamed in the expected output
 RENAMED_NODES = {
   "heredoc_start" => "string",
+  "assign_call"   => "call",
```

Comment:
It might make sense to rename `assign_call` -> `call` in the grammar, I took the quick approach for now.